lldb-eval/parser_test.cc

```diff
@@ -20,8 +20,6 @@
 #include "lldb-eval/expression_context.h"
 #include "lldb/API/SBExecutionContext.h"
 
-// DISALLOW_COPY_AND_ASSIGN is also defined in
-// lldb/lldb-defines.h
```

Comment:
Why removing this comment? We're still including LLDB headers here, so this comment is relevant, no?

Reply:
Oops, I misread it as `lldb-eval/defines.h` 😬